api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -666,9 +683,169 @@ public void shortenedStackTrace_shouldReturnNullIfStackTraceIsNull() {
 	}
 	
 	/**
-	 * @see OpenmrsUtil#nullSafeEqualsIgnoreCase(String,String)
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnTrueWhenFolderContainsFileWithGivenName() {
+		File folder = createFolderInUserDirectory("folder");
+		String separator = FileSystems.getDefault().getSeparator();
```

Comment:
There is alot of code duplication of the above line. Can't we have it as a class constant?

Reply:
ok